Makefile

```diff
@@ -719,6 +719,8 @@ else
 	rm -fr node_modules
 	cp -R $(NODE_MODULES_SRC)/node_modules node_modules
 endif
+  node --version
```

Comment:
we should use $(NODEJS) and $(NPM) which are already defined at [1].

[1] https://github.com/timdream/gaia/blob/node-version/Makefile#L465-L471